modules/rostests/apitests/crt/CMakeLists.txt

```diff
@@ -6,6 +6,6 @@ endif()
 include(ntdll_crt_apitest.cmake)
 include(msvcrt_crt_apitest.cmake)
 
-if(NOT ARCH STREQUAL "amd64")
+if(NOT ARCH STREQUAL "amd64" AND NOT ARCH STREQUAL "arm")
```

Comment:
`ARCH STEQUAL "i386"`?

Reply:
I think this condition is especially for architectures which just don't have working CRT api tests yet.

Reply:
yeah, we have some PPC here and there as well.

Reply:
ARM64?